sso/management/commands/manual_notify_users.py

```diff
@@ -14,43 +17,40 @@ class Command(BaseCommand):
     or not logged in since the aforementioned date.
 
     Usage:
-            python manage.py manual_notify_users 14 --from-year 2018 --from-month 1
-            make manage manual_notify_users -- 14 --from-year 2018 --from-month 1
+            python manage.py manual_notify_users 14 --date 2018-01-01
+            make manage manual_notify_users -- 14 --date 2018-01-01
     """
 
     DATA_RETENTION_STORAGE_YEARS = 3
 
     help = 'Notify inactive users of potential account deletion'
 
     def add_arguments(self, parser):
-        current_datetime = datetime.now()
-        current_year = current_datetime.year
-        current_month = current_datetime.month
-
         parser.add_argument(
             'notification_days',
             type=int,
             help='Number of days of notice to include in the email notification. Possible entries are: 30, 14, 7, 0',
         )
         parser.add_argument(
-            '--from-year',
-            type=int,
-            dest='from_year',
-            default=(current_year - self.DATA_RETENTION_STORAGE_YEARS),
-            help='Year from which to filter inactive user accounts. Defaults to exactly three years back',
-        )
-        parser.add_argument(
-            '--from-month',
+            '--date',
             type=int,
-            dest='from_month',
-            default=current_month,
-            help='Month from which to filter inactive user accounts. Defaults to current month',
+            help='Date from which to filter inactive user accounts with the format YYYY-MM-DD',
         )
 
     def handle(self, *args, **options):
+        now_date = timezone.now()
         notification_days = options['notification_days']
-        from_year = options['from_year']
-        from_month = options['from_month']
+
+        if options['date']:
+            end_date = now_date
+            start_date = timezone.make_aware(
+                datetime.strptime(options['date'], "%Y-%m-%d"), timezone=pytz.timezone(settings.TIME_ZONE)
+            )
+            time_window = relativedelta(start_date, end_date)
+        else:
+            time_window = relativedelta(years=self.DATA_RETENTION_STORAGE_YEARS)
+
+        date = now_date - time_window
```

Comment:
If we know `options['date']` can we not simply set the date to that instead of doing the `time_window`/`relativedelta` routine?

```suggestion
        if options['date']:
            date = timezone.make_aware(
                datetime.strptime(options['date'], "%Y-%m-%d"), timezone=pytz.timezone(settings.TIME_ZONE)
            )
        else:
            date = now_date - relativedelta(years=self.DATA_RETENTION_STORAGE_YEARS)
```